ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/machine/WsAgentStateController.java

```diff
@@ -81,23 +93,22 @@ public WsAgentStateController(WorkspaceServiceClient workspaceServiceClient,
                                   LoaderPresenter loader,
                                   MessageBusProvider messageBusProvider,
                                   AsyncRequestFactory asyncRequestFactory,
-                                  DialogFactory dialogFactory) {
+                                  DialogFactory dialogFactory,
+                                  AppContext appContext) {
         this.workspaceServiceClient = workspaceServiceClient;
         this.loader = loader;
         this.eventBus = eventBus;
         this.messageBusProvider = messageBusProvider;
         this.asyncRequestFactory = asyncRequestFactory;
         this.dialogFactory = dialogFactory;
+        this.appContext = appContext;
         this.availableServices = new ArrayList<>();
     }
 
-    /**
-     * Initialize state of the agent.
-     *
-     * @param devMachine
-     *         development machine instance
-     */
+    /** {@inheritDoc} */
```

Comment:
What is the purpose of an empty javadoc which contains _@inheritDoc_ tag only?
Javadoc tool and all popular IDEs override it automatically when generating documentation.
IMO we should add it only in case of 'real' overriding javadoc like [here](https://github.com/eclipse/che/blob/master/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/wizard/AbstractWizardPage.java#L34).
WDYT?